base/core/src/main/java/org/openscience/cdk/graph/Cycles.java

```diff
@@ -746,7 +746,7 @@ private static Cycles _invoke(CycleFinder finder, IAtomContainer container, int
             /** {@inheritDoc} */
             @Override
             int[][] apply(int[][] graph, int length) throws Intractable {
-                final int threshold = 684; // see. AllRingsFinder.Threshold.Pubchem_99
+                final int threshold = 3072; // see. AllRingsFinder.Threshold.Pubchem_994
```

Comment:
Sure but this may slow things down, it's a setting elsewhere that can be called with the looser value (3072) if needed.